src/main/webapp/index.jsp

```diff
@@ -23,18 +23,20 @@
 
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
-      <% if(request.getSession().getAttribute("user") != null){ %>
-        <a href="/conversations">Conversations</a>
-        <a href="/profiles">Profiles</a>
-
-        <% if(request.getSession().getAttribute("user").equals("EmilyArroyo")||request.getSession().getAttribute("user").equals("AlexandriaStorm")||request.getSession().getAttribute("user").equals("AnaVitoriadoValleCosta")||request.getSession().getAttribute("user").equals("KevinWorkman")||request.getSession().getAttribute("user").equals("GavinLifrieri")) { %>
-          <a href="/admin">Administration</a>
-        <% }else{ %>
-          <a href="/about.jsp">About</a>
-          <a> Hello <%= request.getSession().getAttribute("user") %>!</a>
-          <a href="/activityfeed">Activity</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+      <a href="/conversations">Conversations</a>
+      <a href="/profiles">Profiles</a>
+      <a href="/activityfeed">Activity</a>
+      <%if (request.getSession().getAttribute("user").equals("EmilyArroyo")
```

Comment:
Optional: you could consider extracting the username into a variable and using that for comparison, just to make the code a little easier to read.